package.json

```diff
@@ -10,37 +10,39 @@
     "lint": "vue-cli-service lint",
     "build:lib": "vue-cli-service build --target lib --name Cloudinary src/index.js && npm run bundlewatch",
     "build:styleguide": "rm -rf docs/*; node docs-sources/generateDocsLinks.js; vue-cli-service lint --fix; INTERNAL_DEPS=1 vue-styleguidist build",
-    "storybook:build": "vue-cli-service storybook:build -c config/storybook",
-    "storybook:serve": "vue-cli-service storybook:serve -p 6006 -c config/storybook",
-    "styleguide": "node docs-sources/generateDocsLinks.js; vue-cli-service lint --fix; INTERNAL_DEPS=1 vue-styleguidist server"
+    "styleguide": "node docs-sources/generateDocsLinks.js; vue-cli-service lint --fix; INTERNAL_DEPS=1 vue-styleguidist server",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
   },
   "main": "dist/Cloudinary.umd.js",
   "unpkg": "dist/Cloudinary.umd.min.js",
   "dependencies": {
     "cloudinary-core": "^2.10.3",
-    "cloudinary-video-player": "^1.1.1",
-    "core-js": "2.6.9",
-    "current-script-polyfill": "1.0.0"
+    "current-script-polyfill": "1.0.0",
+    "core-js": "^2.6.9"
   },
   "peerDependencies": {
     "vue": "^2.0.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.9",
```

Comment:
did you test that this doesn't break anything for a real user installing our library?

Reply:
`babel-core` changed to `@babel/core`, will check if it doesn't break anything :|

Reply:
Verified. Just need to install `"@babel/plugin-transform-runtime"` and `"@babel/runtime-corejs2"`, and set the plugin to target corejs 2x in babel.config.js